config/config.yml

```diff
@@ -8,6 +8,8 @@ defaults: &defaults
   ncip_renew_to_agency: "01PRI_INST"
   ncip_renew_application_profile: "SCSB"
   ncip_renew_alma_url: "https://princeton.alma.exlibrisgroup.com/view/NCIPServlet"
+  peoplesoft_bursar_recipients: "cac9@princeton.edu, mzelesky@princeton.edu"
```

Comment:
Could having these emails plaintext in a public repo be an issue?

Reply:
@maxkadel good point.. unfortunately they are already in the file above.  Could we ticket it to make all email lists environment variables ?

Reply:
Sounds good - I tried adding a ticket, please feel free to edit #314